runtime-k3d/src/runtime/controlplane.py

```diff
@@ -120,18 +122,17 @@ def create_cluster(
     else:
         spinner.write("> Creating cluster without proxy configuration.")
 
+    exposed_services_ports = generate_ports_to_expose()
+
     subprocess.check_call(
         [
             "k3d",
             "cluster",
             "create",
             "cluster",
-            "-p",
-            "30555:30555",
-            "-p",
-            "31883:31883",
-            "-p",
-            "30051:30051",
+        ]
+        + exposed_services_ports
+        + [
```

Comment:
```suggestion
        f"{generate_ports_to_expose()},
```

Reply:
does this work? reduces the code, line 125 can be removed than

Reply:
this works with just  + generate_ports_to_expose() + since the function generates an array which will be concatenated